blog/2022-12-01-governance-changes.md

```diff
@@ -0,0 +1,16 @@
+---
+authors:
+  - name: Antonin Delpeuch
+title: "Governance changes: Sandra Fauconnier steps down from project director position to focus on Wikimedia work"
+---
+
+Sandra Fauconnier has been OpenRefine's project director since February 2022, and has been working on many fronts in this capacity: leading a reflection on the transformation of the project's gouvernance model, running the biyearly user survey,
+coordinating our participation in the Outreachy internship programme, migrating from Google Groups to our new Discourse forum, and many more project management tasks behind the scenes.
+Given her close ties with the Wikimedia and Wikibase communities, she also strengthened the project's ties with those partners.
+
+Sandra has decided to step down from the project director role to be able to fully focus on her work on the Wikimedia movement. She plans to still be involved with OpenRefine, for instance to run training workshops about the
```

Comment:
May I suggest to change this to: Sandra has decided to step down from the project director role, returning to her previous work in Linked Open Data in the cultural sector and the Wikimedia movement. She plans to still be involved with OpenRefine's community and will continue to advocate for the tool and to give trainings.

As the Wikimedia grant has stalled, I'm currently having conversations about other types of work (I need to pay my bills). If I end up doing very different things, it will be nice if this blog post doesn't tell lies ;-)